src/lamp_py/ingestion/glides.py

```diff
@@ -244,6 +247,11 @@ def unique_key(self) -> str:
         return "operator"
 
     def convert_records(self) -> pd.Dataset:
+        for record in self.records:
+            record["data"]["signedInAt"] = datetime.fromisoformat(
+                record["data"]["signedInAt"].replace("Z", "+00:00")
+            )
```

Comment:
i have to do this if we want the record to be stored as a pyarrow.timestamp.

would also do a string if you prefer.

Reply:
String probably makes more sense for these, doing the conversion later is pretty inconsequential for such a small dataset